src/Unleash/DefaultUnleash.cs

```diff
@@ -124,14 +125,16 @@ private bool CheckIsEnabled(string toggleName, UnleashContext context, bool defa
             }
             else
             {
-                enabled = featureToggle.Strategies.Any(s => GetStrategyOrUnknown(s.Name).IsEnabled(s.Parameters, enhancedContext, ResolveConstraints(s).Union(s.Constraints)));
+                var strategy = featureToggle.Strategies.FirstOrDefault(s => GetStrategyOrUnknown(s.Name).IsEnabled(s.Parameters, enhancedContext, ResolveConstraints(s).Union(s.Constraints)));
```

Comment:
If there are no strategies. can FirstOrDefault return null and throw NullReferenceException?